modules/ArchiveLibrary/Folder/Repositories/FolderRepository.php

```diff
@@ -189,6 +189,9 @@ public function getFoldersAndFilesByParent(
             if ($parentId != null) {
                 $foldersQuery->where('parent_id', $parentId);
             }
+            else{
```

Comment:
The `else` statement should be on the same line as the closing brace of the `if` statement to follow PSR-12 coding standards.

```suggestion
            } else {
```